src/entry/data-provider/index.ts

```diff
@@ -31,6 +35,32 @@ export const dataProvider = async (request: DataProviderPayload): Promise<DataPr
 
   const trackingBranch = await getTrackingBranchName(groupToken, projectId, defaultBranch);
 
+  const backfillData: BackfillData = {
+    builds: [],
+    deployments: [],
+    metrics: {
+      mrCycleTime: 0,
+      openMergeRequestsCount: 0,
+    },
+  };
+
+  try {
+    const {
+      builds,
+      deployments,
+      metrics: { mrCycleTime, openMergeRequestsCount },
+    } = await getBackfillData('groupToken', projectId, projectName, trackingBranch);
```

Comment:
We're repeating this call on line 68 - we should get rid of that one and then use the data from here.

Reply:
Good catch!